samtranslator/compat.py

```diff
@@ -0,0 +1,8 @@
+try:
+    from pydantic import v1 as pydantic
```

Comment:
If there is no need in keeping v1.8 as a constraint, then there is no need in this `try... except` and `compat.py`